client/src/components/Console.tsx

```diff
@@ -137,107 +137,52 @@ const Console: React.FC<ConsoleProps> = ({ user }) => {
 
     return (
         <main>
-            <Title className="flex items-center">
-                <Link to="/">
-                    {" "}
-                    <span title="Return to Home">
-                        <RiHome2Fill
-                            className="text-black dark:text-white animate-pulse hover:scale-110 hover:shadow-lg transition-transform duration-300 pr-3"
-                            size="32"
-                        />
-                    </span>
-                </Link>
-                Dashboard
-            </Title>
-            <Text>Explore Kiruna</Text>
-            <div className="flex items-stretch mt-6">
+            <div className='flex flex-row mb-0' style={{ marginTop: '-2rem', marginBottom: '2.3rem' }}>
+                <h1 className="text-2xl font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">Dashboard</h1>
```

Comment:
<img width="1440" alt="Screen Shot 2024-12-11 at 10 12 04" src="https://github.com/user-attachments/assets/f5da5231-89da-4644-82e7-e08cea0227b6">

the dashboard, in map section is not responsive.